packages/jest-preset-default/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 4.0.0 (Unreleased)
+
+### Braking Changes
+
+- The bundled `jest` dependency has been updated from requiring `^23.6.0` to requiring `^24.1.0` (see [Braking Changes](https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#breaking-changes), [#13922](https://github.com/WordPress/gutenberg/pull/13922)).
```

Comment:
spelling issue here `Breaking` not `Braking`.  This is in other places too, likely due to copy pasta.

Reply:
Fixed :)